runtime-light/stdlib/string/regex-state.cpp

```diff
@@ -0,0 +1,53 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2024 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include "runtime-light/stdlib/string/regex-state.h"
+
+#include "runtime-common/core/memory-resource/unsynchronized_pool_resource.h"
+#include "runtime-common/core/utils/kphp-assert-core.h"
+#include "runtime-light/k2-platform/k2-api.h"
+#include "runtime-light/state/instance-state.h"
+#include "runtime-light/stdlib/string/regex-include.h"
+
+namespace {
+
+// TODO: use RuntimeAllocator instead
```

Comment:
Would it be fixed in this PR?

Reply:
No. We need to patch our allocator so it can be used in such contexts